src/components/events/Event/index.js

```diff
@@ -45,6 +46,24 @@ module.exports = React.createClass({
     };
   },
 
+  activate: function() {
+    this.setState({ activeClass: 'active' });
+  },
+
+  activateAndTimeout: function() {
+    this.activate();
+    this.timeout();
+  },
+
+  timeout: function() {
+    var _this = this;
```

Comment:
We try to be more semantic when doing this kind of thing. `_this` should be `component`. Makes it more obvious what it is.